extension/llm/export/builder.py

```diff
@@ -162,9 +166,18 @@ def capture_pre_autograd_graph(self) -> "LLMEdgeManager":
         # 2. torch.no_grad() is for getting rid of the dropout (not sure why training ops will show up)
         with torch.nn.attention.sdpa_kernel([SDPBackend.MATH]), torch.no_grad():
             # pyre-fixme[8]
-            self.pre_autograd_graph_module = capture_pre_autograd_graph(
-                self.model, self.example_inputs, dynamic_shapes=dynamic_shape
-            )
+            if self.export_fn == torch.export.export:
+                self.pre_autograd_graph_module = self.export_fn(
+                    self.model,
+                    self.example_inputs,
+                    dynamic_shapes=dynamic_shape,
+                    strict=True,
+                ).module()
+            else:
+                self.pre_autograd_graph_module = self.export_fn(
+                    self.model, self.example_inputs, dynamic_shapes=dynamic_shape
+                )
```

Comment:
Why do you need this if you already patch export_fn

Reply:
And if this is because of differing APIs then I would rather check for qnn backend here instead of using export_fn